oap-server/server-library/library-util/src/main/java/org/apache/skywalking/oap/server/library/util/ResourceUtils.java

```diff
@@ -90,4 +90,13 @@ private static List<File> getDirectoryFilesRecursive(String directoryPath, List<
         }
         return fileList;
     }
+
+    public static File[] getAllPathFiles(String path) throws FileNotFoundException {
```

Comment:
What is the difference from existing `getPathFiles` method?

Reply:
Got it.